ios/blade/Info.plist

```diff
@@ -40,7 +40,13 @@
 		</dict>
 	</dict>
 	<key>NSLocationWhenInUseUsageDescription</key>
-	<string/>
+	<string></string>
+	<key>UIAppFonts</key>
+	<array>
+		<string>Lato-Bold.ttf</string>
+		<string>Lato-Light.ttf</string>
+		<string>Lato-Regular.ttf</string>
+	</array>
```

Comment:
Do the consumers need to add this as well when they use our DS or when they `link` it will be setup automatically?

Reply:
This will be set up automatically. This diff is also XCode rearranging it.